packages/babel/src/transformation/file/options/option-manager.js

```diff
@@ -178,7 +178,7 @@ export default class OptionManager {
     }
 
     // resolve all .babelrc files
-    if (opts.babelrc !== false) {
+    if (opts.babelrc && opts.babelrc.length > 0) {
```

Comment:
just curious, is there any advantage to doing `> 0` explicitly, when the truthiness of `opts.babelrc.length` will be `false` if it's 0 anyway?